### ISSUE TITLE
Stacker News Loot

### ISSUE THREAD

**Issue by kerooke** [stackernews/stacker.news#487]
When Stacker News rewards people, there is no incentive to continue zapping and there is no psychological recognition of a job well done.

Every day we pay 100k+ sats to stackers and hope they choose to pay some of those sats forward to other stackers rather than immediately withdrawing their sats to a cold-storage wallet.

So far, many stackers are choosing to be generous and pay some of their sats forward… but we have no tools to ensure sats spent on Stacker News will circulate throughout the ecosystem moving forward. We also don’t do much to recognize those who are supporting other stackers with their rewards.

This is a problem for a few reasons.

First, the success of Stacker News depends on people using their sats. Without spenders, the Stacker News algorithm won’t be as effective at ranking content. More stackers are spending sats on SN lately, but spenders remain a vanishingly small percentage of all Stacker News visitors. 

Second, our ability to subsidize stackers for their contributions is limited by the leakiness of our rewards “bucket”.

If all the sats we pay to contributors are immediately withdrawn and are never seen again, offering rewards will be a useless exercise.

Third, the direct payment of sats as a reward doesn’t adequately give stackers a feeling of pride or status that they can show off to the rest of the community.

To solve these problems, Stacker News can implement something called “Loot”.

**_At a fundamental level, loot represents zap-locked sats with recognition._** 

Here’s how it might work:

If Stacker News decides to subsidize referrers, creators, or zappers, loot can be added to the daily sats rewards of top contributors. All rewards, whether in the form of loot or sats should only be available for those with cowboy hats.

Loot is only useful if shared, and must be forwarded to another stacker in order to claim the sats associated with it.

In other words, it can’t immediately be withdrawn by the stacker. When a stacker shares their loot with another stacker, they each receive 50% of the sats that had been locked in their loot (nobody knows how many sats are locked in loot until this moment).

Both participants also get a badge on their profile showing their loot collection. Unlike cowboy hats, there is no streak involved, it’s just a sum of all loot collected. These icons could also be displayed on the post or comment of the loot forwardee. 

Loot could be expressed as gold, silver, copper, cattle, bison, stagecoaches, guns, or any other valuable item in the old west.

As an example, if I am deemed to be worthy of loot, I might see a notification saying something like:

“Yesterday you went to explore the foothills outside of town, and stumbled upon a broken-down stagecoach.

Take another stacker to your stagecoach so they can help you fix it up and sell it for sats.”

Then I would see a stagecoach icon appear when I long-press the zap button, and clicking it will share my stagecoach with the zap recipient.

The resulting notification could be something like this:  “You and @insertusername fixed up your stagecoach and sold it for 1,300 sats each. Nice work!”

As a safeguard against bots or fake accounts, this stagecoach reward doesn’t always have to be paid. Stacker News could set thresholds around how many sats the loot forwardee has earned, how long their account has been active, or how many other people have zapped their specific post or comment.

In some cases, sharing a stagecoach may lead to a notification like this:  “You tried your best to fix up the stagecoach, but a dust storm blew in and buried it in the sand. Better luck next time.”

Ultimately, I believe this will lead to normalizing the behavior of zapping, it will introduce collectibles for stackers to show off on their profile, and it will enhance the community relationships on the site.

It will also make it harder, though not impossible, for stackers to immediately withdraw all rewards without paying some of them forward to the rest of the community.

**Comment by hellmanhellman** [comment]
If I understand correctly, you propose we introduce a kind of "gift card" that can only be spent on stacker.news, in order to deter users from not being generous enough with their zapping? Let me know if this summary is accurate.

I'm not sure about your premise. Yes, rewards go out, but is that why people use stacker? I don't know about you, but I definitely don't care about the few dollars gained or lost, **the value of stacker.news is the community itself**. 
The fact that I can zap someone 500 sats ($0.14) is a symbolic gesture, not financial support. 

> we have no tools to ensure sats spent on Stacker News will circulate throughout the ecosystem moving forward

Why would that be needed? Imposing rules and systems on users in order to prevent them from doing _whatever they want with their money_ will help no one, but would instead make people reluctant to fund their stacker account in the first place.

> First, the success of Stacker News depends on people using their sats. Without spenders, the Stacker News algorithm won’t be as effective at ranking content. 

How can content be effectively ranked, if a bias is put in place that favors spamming ("keeping your cowboy hat") as well as random zapping ("ugh, my sats are locked up in loot, i guess ill have to zap some random posts to get it out") ?

I like parts of the loot idea though, but in the form of awards. Like, pay a little extra and you can award a comment with a heart emoji, for instance. And the loot idea also reminds me a bit of having achievements, that's fun.

So tl;dr: This sounds to me like a solution without a problem. Let me know what you think.
